SOURCE/InputOutput.c

```diff
@@ -1151,18 +1140,17 @@ void ReadInputFile( char* fin_name, int *istep, int *irestart, int *writer, int
     model->user6           = ReadDou2( fin, "user6",           0.0 );
     model->user7           = ReadDou2( fin, "user7",           0.0 );
     model->user8           = ReadDou2( fin, "user8",           0.0 );
-
     // Derived quantities
     model->dx                = (model->xmax - model->xmin) / (model->Nx - 1);
     model->dz                = (model->zmax - model->zmin) / (model->Nz - 1);
     model->dt0               = model->dt;
     model->dt_start          = model->dt;
     model->dt_max            = ReadDou2( fin, "dt_max",     1e20 ) / scaling->t; // maximum allowed time step
```

Comment:
Could you please also add few lines why do we need a fixed dt_max?

Reply:
done